source/_components/fronius.markdown

```diff
@@ -0,0 +1,104 @@
+---
+layout: page
+title: "Fronius"
+description: "Instructions on how to connect your Fronius Inverter to Home Assistant."
+date: 2019-06-20 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category:
+  - Energy
+  - Sensor
+logo: fronius.png
+ha_iot_class: Local Polling
+ha_release: 0.96
+---
+
+The `fronius` sensor will poll a [Fronius](http://www.fronius.com/) solar inverter, battery system or smart meter and present the values as sensors (or attributes of sensors) in Home Assistant.
+
+## {% linkable_title Configuration %}
+
+To enable this sensor, add the following lines to your `configuration.yaml` file:
+
+```yaml
+sensor:
+  - platform: fronius
+    resource: FRONIUS_IP_ADRESS
```

Comment:
As this needs to be the URL it would be better to say `FRONIUS_URL` than ipaddress (2 d in ipaddress).

Reply:
That's a good point, will include it :)